tests/test_it.py

```diff
@@ -279,7 +279,7 @@ def test_request_make_body_seekable_cleans_up_threadmanger_on_exception(
     assert len(manager.stack) == 0
     app = config.make_wsgi_app()
     app = webtest.TestApp(app)
-    with pytest.raises(Exception):
+    with pytest.raises(IOError):
```

Comment:
I selected `IOError` here as that's the ancestor for [`webob.request.DisconnectionError`](https://github.com/Pylons/webob/blob/39d5af3c797e7b867f152c2e8c979de42d029403/src/webob/request.py#L1607)